data_registry/process_manager/task/process.py

```diff
@@ -38,6 +40,20 @@ def get_status(self):
             self.job.context["process_data_version"] = compile_releases.get("data_version")
             self.job.save()
 
+        if is_last_completed:
+            response = request(
+                "GET",
+                urljoin(settings.KINGFISHER_PROCESS_URL, f"{base_url}/{self.process_id}/metadata/"),
```

Comment:
Hmm, you wrote this endpoint to accept the compiled collection, rather than the base collection. The compiled collection is stored in `compile_releases`.

Reply:
Indeed. `process_id` is set to the original collection by the collect task (extracted from the crawl log). Then `process_id_pelican` is set to the compiled collection by the process task (because Pelican operates on compiled releases).